enterprise/models.py

```diff
@@ -1322,9 +1322,12 @@ def contains_courses(self, content_ids):
         catalog_client = get_course_catalog_api_service_client(self.enterprise_customer.site)
         course_keys = {catalog_client.get_course_id(k) for k in content_ids}
 
+        # Remove `None` from set of course_keys if present
+        course_keys = course_keys.difference({None})
+
         content_ids_in_catalog = self.content_filter_ids
         if not content_ids_in_catalog:
-            content_ids_in_catalog = self._filter_members('key', list(course_keys))
+            content_ids_in_catalog = self._filter_members('key', list(course_keys)) if course_keys else []
```

Comment:
`_filter_members()` returns a set not a list, and we use set methods on it below, so we need `else set()` instead of `else []`

Reply:
good catch! thanks!